models/basgra/tests/testthat.R

```diff
@@ -0,0 +1,13 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+library(testthat)
+library(PEcAn.utils)
+
+PEcAn.logger::logger.setQuitOnSevere(FALSE)
+#test_check("PEcAn.ModelName")
```

Comment:
We probably _do_ want to run these tests, right? I would enable this now before we forget (especially since there are no actual tests that will be run inside this package right now).

```suggestion
test_check("PEcAn.BASGRA")
```